packages/director/package.json

```diff
@@ -24,7 +24,10 @@
     "clean": "rimraf ./dist"
   },
   "dependencies": {
+    "@types/minio": "^7.0.5",
```

Comment:
This is a dev dep

Reply:
Im pretty sure this was an artifact of a rebase I did on the v5 branch. Ill move it.